Seed data into the destination databases (#368)
bruin-data/bruin: docs/assets/sql-seed.md

> # SQL Seed Assets

Reviewer: I wouldn't call it `SQL Seed Assets`, simply call them `Seed Assets`.
```suggestion
# Seed Assets
```

> Bruin supports running SQL assets against a variety of data platforms natively. Along with SQL assets, Bruin also supports running SQL Seed assets that will populate a table with data from a csv file. 
> 

Reviewer: ```suggestion
Seeds are CSV-files that contain data that is prepared outside of your pipeline that will be loaded into your data platform. Bruin supports seed assets natively, allowing you to simply drop a CSV file in your pipeline and ensuring the data is loaded to the destination platform accurately.

Seed assets are defined with `<platform>.seed` asset types, e.g.:
- for BigQuery: `bq.seed`
- for DuckDB: `duckdb.seed`
```

> You can define SQL Seed assets in a file ending with `.yaml`:

Reviewer: ```suggestion
You can define seed assets in a file ending with `.yaml`:
```

Reviewer: I think we should also add more details about the `path` parameter, since it is supposed to be optional.
